docs/guides/agent/agent_component_reference/begin.md

```
@@ -5,122 +5,399 @@ sidebar_custom_props: {
+Click the plus sign next to any component on the canvas to select the next component. Common components include:
+`Begin`, `Agent`, `Retrieval`, `Message`, `Await response`, `Switch`, `Iteration`, `Categorize`, `Code`, `Text processing`, `Execute SQL`, `HTTP Request`, and pipeline-related components: `Parser`, `Title chunker`, `Token chunker`, `Transformer`, `Indexer`.
+
+After adding a component, click the component itself to open the configuration panel on the right. Fields in the configuration panel define input data, processing logic, output variables, and references for subsequent steps.
+
+### Use Variables
+Components on the Agent canvas support variable references to implement data transmission between components. Variable sources include system variables, global variables defined in the Begin component, and outputs from upstream components.
+
+In input boxes supporting variable references, type `/` or click the variable button next to the input box to open the variable selector.
+
+Common variables:
+| Variable | Description |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)